docs/data-sources/infoblox_mx_record.md

```diff
@@ -0,0 +1,40 @@
+# MX-record Data Source
+
+Use the data source to retrieve the following information for an MX-record from the corresponding object in NIOS:
+
+* `zone`: the zone which the record belongs to.
+* `ttl`: the "time to live" value of the record, in seconds. Example: `1800`.
+* `comment`: the description of the record. This is a regular comment. Example: `spare node for the service`.
+* `ext_attrs`: the set of extensible attributes of the record, if any. The content is formatted as a JSON map. Example: `{"Owner”: "State Library”, "Expires”: "never”}`.
+
+The following list describes the parameters you must define in an `infoblox_mx_record` data source block:
+
+* `dns_view`: optional, specifies the DNS view in which the reverse mapping zone exists. If a value is not specified, the name `default` is used as the DNS view.
+* `fqdn`: required, required, specifies the fully qualified domain name which a mail exchange host is assigned to. Example: `big-big-company.com`
```

Comment:
The word 'required' is repeating. The description needs to be updated. It is not for a mail exchange host.